http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/JsonJacksonPayloadIT.java

```diff
@@ -15,7 +15,8 @@ public class JsonJacksonPayloadIT extends JsonPayloadIT {
     private static final String REST_JACKSON = "rest-jackson";
 
     @QuarkusApplication(classes = { FootballTeamResource.class, FootballTeam.class, Person.class,
-            PersonResource.class }, properties = "oidcdisable.properties", dependencies = @Dependency(artifactId = "quarkus-"
+            PersonResource.class,
+            JsonExceptionMapper.class }, properties = "oidcdisable.properties", dependencies = @Dependency(artifactId = "quarkus-"
```

Comment:
Afaik I know it's draft and you still looking at this but have you been able to look how we can check what extension is used for serialization/deserialization? As that was one of the main point of discussion on previous PR.

If we can't determine which extension is used wee need to discuss what we can do or if the TP should not be updated taht we testing only one extension.

Reply:
Currenlty I will test only with JsonB library, so I guess this comment is resolved.